flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackend.java

```diff
@@ -104,10 +107,8 @@
 	@Nullable
 	private OptionsFactory optionsFactory;
 
-	/** True if incremental checkpointing is enabled.
-	 * Null if not yet set, in which case the configuration values will be used. */
-	@Nullable
-	private Boolean enableIncrementalCheckpointing;
+	/** True if incremental checkpointing is enabled. */
+	private TernaryBoolean enableIncrementalCheckpointing;
```

Comment:
what about renaming it to `isIncrementalCheckpointingEnabled`. This makes it clearer that this is a boolean.

Reply:
I think by convention, the `is` should not be part of a boolean field name, only replace the `get` in the getter.